docs/source/deployment/prefect.md

```diff
@@ -113,8 +92,124 @@ def build_and_register_flow(pipeline_name, env):
     flow.run_agent()
 
 
+class KedroInitTask(Task):
+    """Task to initialize kedro session"""
```

Comment:
```suggestion
    """Task to initialize KedroSession"""
```

Reply:
Noticed that I haven't set up github to sign off on suggest changes I commit through the web interface. Will solve that problem some other time. For now I am implementing all your suggestions locally to allow for sign off.